lisa/operating_system.py

```diff
@@ -17,6 +19,18 @@
 _get_init_logger = partial(get_logger, name="os")
 
 
+@dataclass
+class OsVersion:
+    vendor: str
+    release: str = ""
+    codename: str = ""
+    package: str = ""
+    update: str = ""
```

Comment:
What's those fields refer to? Can yo provide a link to learn?

Reply:
> What's those fields refer to? Can yo provide a link to learn?

sure, I will add the link. With these, we can get all the info about the vendor, release, the type of package it support and the codename for the distro version.

Reply:
Please provide the link for reference.

Reply:
In LISAv2, we had GetOsVersion() which kind of fetch similar info for different distros.
Added the links

Reply:
The "package" looks not used in v2, and we have another way in v3. Please remove it. For others. it's ok to keep them.

Reply:
Done